src/helpers.rs

```diff
@@ -1187,6 +1187,58 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
 
         interp_ok(array)
     }
+
+    fn check_fn_abi(
+        &mut self,
+        input_args: &[Ty<'tcx>],
+        output_ty: Ty<'tcx>,
+        safety: Safety,
+        abi: ExternAbi,
```

Comment:
This should probably go before `input_args`, also to match the order in which these things show up in the Rust source code (`extern "ABI" fn(input) -> output`).